packages/terra-form-select/src/_Menu.jsx

```diff
@@ -126,20 +149,41 @@ class Menu extends React.Component {
 
   componentDidMount() {
     document.addEventListener('keydown', this.handleKeyDown);
+    /**
+     * Without this detection for ontouchstart and the early return, VoiceOver on iOS will read the
+     * first option twice when the menu is opened. First due to aria-live update in componentDidMount
+     * and another time because we shift focus to the dropdown and VoiceOver on iOS will read the
+     * first interactable element in the dropdown. To mitigate this, the following conditional
+     * check opts-out of the aria-live update if browser supports ontouchstart which iOS supports.
+     */
+    if ('ontouchstart' in window) return;
+    this.updateCurrentActiveScreenReader();
   }
 
   componentDidUpdate() {
-    this.updateLiveRegion();
+    this.updateNoResultsScreenReader();
     this.scrollIntoView();
   }
 
   componentWillUnmount() {
     this.clearSearch();
     this.clearScrollTimeout();
     document.removeEventListener('keydown', this.handleKeyDown);
+
+    if (this.props.focusRegion) {
+      this.props.focusRegion.removeAttribute('data-activedescendant');
```

Comment:
Where is data-activedescendant used? Should this be aria-activedescendant?

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2236/commits/7ed989860674b09c4c4c49b168d745eb37edc8f5